### PR TITLE
Issue 204 - Update for Master (rarnold-performance)

### PR DESCRIPTION
**Pull request by prb112**

### CODE REVIEW

**Comment by prb112** [comment]
All Tests pass.  DCO is not something I can update as another commit was layered ontop.

**Comment by albertwang-ibm** [comment]
DCO check failed.

**Comment by albertwang-ibm** [comment]
the codes looks good for me

**Comment by prb112** [comment]
> DCO check failed.

Correct, I'll manually set to passed.  There are layers of commits on top of the failed DCO.

File: fhir-model/src/test/java/com/ibm/fhir/model/spec/test/Analytics.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author rarnold
+ *
+ */
```
Comment (lmsurpre) [inline fhir-model/src/test/java/com/ibm/fhir/model/spec/test/Analytics.java]: prefer to remove this

File: fhir-model/src/test/java/com/ibm/fhir/model/spec/test/AnalyticsTest.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import static org.testng.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+/**
+ * Simple unit test of the analytics
+ * @author rarnold
```
Comment (lmsurpre) [inline fhir-model/src/test/java/com/ibm/fhir/model/spec/test/AnalyticsTest.java]: remove `@author`

File: fhir-model/src/test/java/com/ibm/fhir/model/spec/test/DriverMetrics.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.model.spec.test;
+
+/**
+ * Used to collect response time metrics during development
+ * (e.h. test-case runs) to identify hotspots or concurrency issues. 
+ * Not intended for runtime instrumentation.
+ * @author rarnold
```
Comment (lmsurpre) [inline fhir-model/src/test/java/com/ibm/fhir/model/spec/test/DriverMetrics.java]: same

File: fhir-server-test/src/test/java/com/ibm/fhir/server/test/ExampleRequestProcessor.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-/*
+/**
```
Comment (lmsurpre) [inline fhir-server-test/src/test/java/com/ibm/fhir/server/test/ExampleRequestProcessor.java]: this is non-javadoc so the single `*` is correct / consistent with the rest of our files

**Comment by prb112** [comment]
> very minor, but for consistency with rest of project it would be good to:
> 
>     1. change `/**` to `/*` in the copyright header
> 
>     2. remove the `@author` tag in javadoc

Yeah - I intentionally did not do so as I had the other PR open which would conflict with those changes. 

I can make those changes.

File: fhir-model/src/test/java/com/ibm/fhir/model/spec/test/Analytics.java (hunk 1 above)
Comment (prb112) [reply fhir-model/src/test/java/com/ibm/fhir/model/spec/test/Analytics.java]: same... reason is below

**Comment by prb112** [comment]
Updated per Lee's comments

**Pull request merged**